File: src/main/java/land/leets/domain/comment/domain/repository/CommentRepository.java
package land.leets.domain.comment.domain.repository;

import land.leets.domain.comment.domain.Comment;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CommentRepository extends JpaRepository<Comment, Long> {
    List<Comment> findAllByApplicationId(Long applicationId);
}

File: src/main/kotlin/land/leets/domain/comment/domain/repository/CommentRepository.kt
package land.leets.domain.comment.domain.repository

import land.leets.domain.comment.domain.Comment
import org.springframework.data.jpa.repository.JpaRepository

interface CommentRepository : JpaRepository<Comment, Long> {
    fun findAllByApplicationId(applicationId: Long): List<Comment>
}
